### PR TITLE
Polish printer at Ubuntu

### PR DESCRIPTION
**Pull request by core-man**

### CODE REVIEW

File: source/ntu/printer/index.rst [hunk 1]
```diff
@@ -60,13 +59,19 @@ For Windows, it is the same as the above manual, except the following parts:
...
-5. Select drivers, now it is ``HP-LaserJetPro-M404-M405``.
+
+1.  `Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>`__ first if using wired connection
+2.  Install prerequisite::
+
+    $ sudo apt install samba
+    $ sudo apt install smbclient
+
+3.  Setting: ``Settings`` -> ``Printers`` -> ``Additional Printer Settings`` ->
+    ``Add`` -> ``Network Printer`` -> ``Windows Printer Via SAMBA``
```
Comment (core-man) [inline source/ntu/printer/index.rst:72]:
```suggestion
3.  Setting

    ``Settings`` -> ``Printers`` -> ``Additional Printer Settings`` ->
    ``Add`` -> ``Network Printer`` -> ``Windows Printer Via SAMBA``
```

File: source/ntu/printer/index.rst [hunk 2]
```diff
@@ -60,13 +59,19 @@ For Windows, it is the same as the above manual, except the following parts:
...
+4. Input link (``smb://mps.ntu.edu.sg/SPMS-MAS-04-07-M01``), user name (``domain/username``) and password
```
Comment (core-man) [inline source/ntu/printer/index.rst:73]:
```suggestion
4. Input

    **Link**: ``smb://mps.ntu.edu.sg/SPMS-MAS-04-07-M01``
    **User name**: ``domain/username``
    **Password**
```

File: source/ntu/printer/index.rst [hunk 3]
```diff
@@ -60,13 +59,25 @@ For Windows, it is the same as the above manual, except the following parts:
...
+3.  Setting
+
+    ``Settings`` -> ``Printers`` -> ``Additional Printer Settings`` ->
+    ``Add`` -> ``Network Printer`` -> ``Windows Printer Via SAMBA``
+4. Input
+
+    **Link**: ``smb://mps.ntu.edu.sg/SPMS-MAS-04-07-M01``
+    **User name**: ``domain/username``
+    **Password**
+5. Select drivers (``HP-LaserJetPro-M404-M405``)
```
Comment (core-man) [inline source/ntu/printer/index.rst:80]:
```suggestion

5. Select drivers (``HP-LaserJetPro-M404-M405``)
```

File: source/ntu/printer/index.rst [hunk 4]
```diff
@@ -60,13 +59,26 @@ For Windows, it is the same as the above manual, except the following parts:
...
+1.  `Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>`__ first if using wired connection
+2.  Install prerequisite::
+
+    $ sudo apt install samba
+    $ sudo apt install smbclient
+
+3.  Setting
+
+    ``Settings`` -> ``Printers`` -> ``Additional Printer Settings`` ->
+    ``Add`` -> ``Network Printer`` -> ``Windows Printer Via SAMBA``
```
Comment (core-man) [inline source/ntu/printer/index.rst:74]:
```suggestion
    ``Add`` -> ``Network Printer`` -> ``Windows Printer Via SAMBA``

```

File: source/ntu/printer/index.rst [hunk 5]
```diff
@@ -60,13 +59,27 @@ For Windows, it is the same as the above manual, except the following parts:
...
+
+4. Input
+
+    **Link**: ``smb://mps.ntu.edu.sg/SPMS-MAS-04-07-M01``
+    **User name**: ``domain/username``
+    **Password**
+
+5. Select drivers (``HP-LaserJetPro-M404-M405``)
```
Comment (core-man) [inline source/ntu/printer/index.rst:82]:
```suggestion
5. Select drivers: ``HP-LaserJetPro-M404-M405``
```

File: source/ntu/printer/index.rst [hunk 6]
```diff
@@ -60,13 +59,27 @@ For Windows, it is the same as the above manual, except the following parts:
...
+
+    $ sudo apt install samba
+    $ sudo apt install smbclient
+
+3.  Setting
+
+    ``Settings`` -> ``Printers`` -> ``Additional Printer Settings`` ->
+    ``Add`` -> ``Network Printer`` -> ``Windows Printer Via SAMBA``
+
+4. Input
```
Comment (core-man) [inline source/ntu/printer/index.rst:76]:
```suggestion
4. Input:
```

File: source/ntu/printer/index.rst [hunk 7]
```diff
@@ -60,13 +59,27 @@ For Windows, it is the same as the above manual, except the following parts:
...
-4. Input link ``smb://mps.ntu.edu.sg/SPMS-MAS-04-07-M01``, User name ``domain/username`` and password.
-5. Select drivers, now it is ``HP-LaserJetPro-M404-M405``.
+
+1.  `Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>`__ first if using wired connection
+2.  Install prerequisite::
+
+    $ sudo apt install samba
+    $ sudo apt install smbclient
+
+3.  Setting
```
Comment (core-man) [inline source/ntu/printer/index.rst:71]:
```suggestion
3.  Setting:
```

File: source/ntu/printer/index.rst [hunk 8]
```diff
@@ -60,13 +59,27 @@ For Windows, it is the same as the above manual, except the following parts:
...
+3.  Setting:
+
+    ``Settings`` -> ``Printers`` -> ``Additional Printer Settings`` ->
+    ``Add`` -> ``Network Printer`` -> ``Windows Printer Via SAMBA``
+
+4. Input:
+
+    **Link**: ``smb://mps.ntu.edu.sg/SPMS-MAS-04-07-M01``
+    **User name**: ``domain/username``
+    **Password**
```
Comment (core-man) [inline source/ntu/printer/index.rst:80]:
```suggestion
    - **Link**: ``smb://mps.ntu.edu.sg/SPMS-MAS-04-07-M01``
    - **User name**: ``domain/username``
    - **Password**
```

File: source/ntu/printer/index.rst [hunk 9]
```diff
@@ -60,13 +59,27 @@ For Windows, it is the same as the above manual, except the following parts:
...
+1.  `Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>`__ first if using wired connection
+2.  Install prerequisite::
+
+    $ sudo apt install samba
+    $ sudo apt install smbclient
+
+3.  Setting:
+
+    ``Settings`` -> ``Printers`` -> ``Additional Printer Settings`` ->
+    ``Add`` -> ``Network Printer`` -> ``Windows Printer Via SAMBA``
```
Comment (core-man) [inline source/ntu/printer/index.rst:74]:
```suggestion
    ``Settings`` -> ``Printers`` -> ``Additional Printer Settings``
     -> ``Add`` -> ``Network Printer`` -> ``Windows Printer Via SAMBA``
```

File: source/ntu/printer/index.rst [hunk 10]
```diff
@@ -60,13 +59,27 @@ For Windows, it is the same as the above manual, except the following parts:
...
+1.  `Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>`__ first if using wired connection
+2.  Install prerequisite::
+
+    $ sudo apt install samba
+    $ sudo apt install smbclient
+
+3.  Setting:
+
+    ``Settings`` -> ``Printers`` -> ``Additional Printer Settings``
+     -> ``Add`` -> ``Network Printer`` -> ``Windows Printer Via SAMBA``
```
Comment (core-man) [inline source/ntu/printer/index.rst:74]:
```suggestion
    ``Settings`` -> ``Printers`` -> ``Additional Printer Settings`` ->
    ``Add`` -> ``Network Printer`` -> ``Windows Printer Via SAMBA``
```

**Comment by core-man** [comment]
@HouseJaay See new style: https://migg-ntu.github.io/sitepreview/MIGG-NTU/MIG_Docs/printer-at-Ubuntu/ntu/printer/#using-printer-at-ubuntu

**Review by HouseJaay — APPROVED** [review]

**Pull request merged**